README.md

```diff
@@ -1,27 +1,31 @@
 # prox
 
-**Installation**
+## Installation
+Install Cocoapods. If you already have Cocoapods installed, run the same
+command to update:
 
-Install Cocoapods. If you already have Cocoapods installed, run the same command to update
+    sudo gem install cocoapods
 
-```
-sudo gem install cocoapods
-```
+If it has been a while since you last updated Cocoapods, you may want to update
+your pods spec repo (this can take some time):
 
-If it has been a while since you last updated Cocoapods, you may want to update your pods spec repo (this can take some time)
+    pod repo update
 
-```
-pod repo update
-```
+Install Pods:
 
-Install Pods
+    pod install
 
-```
-pod install
-```
+Open the workspace. You will need to use the workspace to do development from
+now on. Attempting to compile the .xcodeproj by itself will result in
+compilation failures:
 
-Open the workspace. You will need to use the workspace to do development from now on. Attempting to compile the .xcodeproj by itself will result in compilation failures.
+    open Prox.xcworkspace
 
-```
-open Prox.xcworkspace
-```
\ No newline at end of file
+### GoogleServices-Info.plist
+For a successful build, you need the `Prox/Prox/GoogleService-Info.plist` file
+which contains secrets to connect to Firebase and our backend services. *Please
```

Comment:
I'd call it 'secure information' rather than 'secrets'

So:

`which contains _secure information_ to connect to Firebase and our backend services`